src/main/webapp/file-info-fragment.xhtml

```diff
@@ -10,52 +10,79 @@
     xmlns:cc="http://java.sun.com/jsf/composite"
     xmlns:o="http://omnifaces.org/ui"
     xmlns:iqbs="http://xmlns.jcp.org/jsf/composite/iqbs">
-    <p:outputPanel>
-        <div id="fileInfoInclude-filesTable" class="col-sm-12 row">
-            <div class="pull-left col-file-thumb">
-                <div class="thumbnail-block text-center">  
-                    <span class="icon-#{dataFileServiceBean.getFileThumbnailClass(fileMetadata.dataFile)} file-thumbnail-icon text-muted" jsf:rendered="#{!EditDatafilesPage.isThumbnailAvailable(fileMetadata)}"/>
-                    <span class="file-thumbnail-preview-img" jsf:rendered="#{EditDatafilesPage.isThumbnailAvailable(fileMetadata)}"
-                          data-toggle="popover" data-placement="bottom" data-trigger="hover" data-html="true" data-content="&lt;img src=&#34;/api/access/datafile/#{fileMetadata.dataFile.id}?imageThumb=400&#34; alt=&#34;#{bundle['file.preview']} #{fileMetadata.label}&#34; /&gt;"
-                          data-template='&lt;div class="popover thumb-preview" role="tooltip"&gt;&lt;div class="arrow"&gt;&lt;/div&gt;&lt;h3 class="popover-title"&gt;&lt;/h3&gt;&lt;div class="popover-content"&gt;&lt;/div&gt;&lt;/div&gt;'>
-                        <p:graphicImage value="/api/access/datafile/#{fileMetadata.dataFile.id}?imageThumb=true" alt="#{fileMetadata.label}"/>
-                    </span>
-                    <!-- Restricted File Icon -->
-                    <div class="file-icon-restricted-block" jsf:rendered="#{fileMetadata.restricted and !fileDownloadHelper.canDownloadFile(fileMetadata)}">                  
-                        <span class="glyphicon glyphicon-lock text-danger"/>
-                    </div>
-                    <div class="file-icon-restricted-block" jsf:rendered="#{fileMetadata.restricted and fileDownloadHelper.canDownloadFile(fileMetadata) }">
-                        <span class="icon-unlock text-success"/>
-                    </div>
+    <p:outputPanel id="fileInfoInclude-filesTable" styleClass="media">
+        <div class="media-left col-file-thumb" style="padding-top:4px;">
+            <div class="media-object thumbnail-block text-center">
+                <span class="icon-#{dataFileServiceBean.getFileThumbnailClass(fileMetadata.dataFile)} file-thumbnail-icon text-muted" jsf:rendered="#{!dataFileServiceBean.isThumbnailAvailable(fileMetadata.dataFile)}"/>
```

Comment:
@scolapasta I believe this is change related to the removing of the isThumbnailAvailable function from EditDatafilesPage.java that you commented on, as I am now using this include on more than just the edit file pgs, so I am pointing the logic at the dataFileServiceBean version of the logic.